README.md

```diff
@@ -0,0 +1,607 @@
+# Tariff Interpreter
+
+___
+This Interpreter is a reverence implementation for computing a JSON formatted bike-sharing tariff model.
+
+## Contents
+
+___
+
+* [Why this interpreter](#why-this-tariff-schema)
+* [General Interpreter Operation](#general-interpreter-operation)
+* [Tariff JSON Schemas](#tariff-json-schema)
+
+## Why this Tariff-Schema
```

Comment:
I'd suggest to remove this section. It's not helpful for outsiders.

Reply:
Also, please refrain from using the first person too much "we" and write it more formal / from an outside perspective (if applicable to other paragraphs.